platform-server-rest/src/main/java/uno/cod/platform/server/rest/controller/ChallengeController.java

```diff
@@ -21,17 +22,20 @@ public ChallengeController(ChallengeService service) {
     }
 
     @RequestMapping(value = RestUrls.CHALLENGES, method = RequestMethod.POST)
+    @PreAuthorize("isAuthenticated() and @securityService.isOrganizationAdmin(principal, #dto.organizationId)")
```

Comment:
On what grounds did you define that? Why should only admins list/create challenges?